app/src/main/java/ar/com/wolox/android/training/ui/training/main/tabs/news/NewsAdapter.kt

```diff
@@ -2,60 +2,60 @@ package ar.com.wolox.android.training.ui.training.main.tabs.news
 
 import android.net.Uri
 import android.view.LayoutInflater
+import android.view.View
 import android.view.ViewGroup
 import android.widget.ImageView
 import android.widget.TextView
 import androidx.recyclerview.widget.RecyclerView
 import ar.com.wolox.android.R
 import ar.com.wolox.android.training.model.NewsItem
 import com.facebook.drawee.view.SimpleDraweeView
+import kotlinx.android.synthetic.main.news_item.view.*
 import org.ocpsoft.prettytime.PrettyTime
 import java.util.Locale
 
 class NewsAdapter(private val dataSet: List<NewsItem>, private val clickListener: (NewsItem) -> Unit) : RecyclerView.Adapter<NewsAdapter.NewsViewHolder>() {
 
-    class NewsViewHolder(inflater: LayoutInflater, parent: ViewGroup) :
-            RecyclerView.ViewHolder(inflater.inflate(R.layout.news_item, parent, false)) {
-        private var mUserView: TextView? = null
-        private var mNewsView: TextView? = null
-        private var mUserIcon: SimpleDraweeView? = null
-        private var mLikeBtn: ImageView? = null
-        private var mDate: TextView? = null
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): NewsViewHolder {
+        return NewsViewHolder(LayoutInflater.from(parent.context).inflate(R.layout.news_item, parent, false))
+    }
+
+    override fun onBindViewHolder(holder: NewsViewHolder, position: Int) {
+        val news: NewsItem = dataSet[position]
+        holder.bind(news, clickListener)
+    }
+
+    override fun getItemCount(): Int = dataSet.size
+
+    class NewsViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
+        private var vUsername: TextView? = null
```

Comment:
After commenting this, I don't think this variables are necessary, you could just call `itemView.vUsername` when you need it.